packages/frontend/src/admin-pages/components/header/Navigation.vue

```diff
@@ -158,19 +158,19 @@ export default {
       event.preventDefault()
       const vue = this
       if (vue.displayingMenu) {
-        $(vue.$refs.oMenubarPopupParent).removeClass('o-menubar-popup-parent--active')
-        $(vue.$refs.oMenubarPopup).fadeOut(200)
+        window.$(vue.$refs.oMenubarPopupParent).removeClass('o-menubar-popup-parent--active')
```

Comment:
I don't think the `window.` is needed at all in this page

Reply:
I removed those